package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jspm-tsc-update",
-  "version": "0.1.2",
+  "version": "0.2.0-beta",
```

Comment:
Can you please revert the version number? I will then bump it alongside releasing a new version. Thanks!